docs/concepts-broker-load-balancing-types.md

```diff
@@ -0,0 +1,63 @@
+---
+id: concepts-broker-load-balancing-types
+title: Types
+sidebar_label: "Types"
+---
+
+Pulsar provides three types of broker load balancers:
+
+- Simple broker load balancer: it existed since Pulsar's inception, but now is deprecated. No one uses it.
+
+- [Modular broker load balancer](#side-by-side-comparisons)
+
+- [Extensible broker load balancer](#side-by-side-comparisons)
+
+The modular and extensible broker load balancer use similar implementation logic and algorithm and have similar functionalities. For example, they both employ a similar approach to distributing data loads among brokers, including:
+
+- Dynamic [bundle-broker assignment](./concepts-broker-load-balancing-concepts.md#bundle-assignment)
+
+- Dynamic [bundle splitting](./concepts-broker-load-balancing-concepts.md#bundle-splitting)
+
+- Dynamic [bundle unloading (shedding)](./concepts-broker-load-balancing-concepts.md#bundle-unloading)  
+
+As a result, brokers dynamically collect and share the current load with other brokers for better load balancing decisions. 
```

Comment:
```suggestion
As a result, brokers dynamically collect and share the current load with other brokers for better load-balancing decisions. 
```

Reply:
I think both "load balancing decisions" and "load-balancing decisions" are all correct.